INSTALLATION.md

```diff
@@ -31,9 +31,9 @@ Containers:       cilium             Running: 2
                   cilium-operator    Running: 1
                   hubble-relay       Running: 1
 Cluster Pods:     3/3 managed by Cilium
-Image versions    cilium-operator    quay.io/cilium/operator-aws-service-mesh:v1.11.0-beta.1: 1
-                  hubble-relay       quay.io/cilium/hubble-relay-service-mesh:v1.11.0-beta.1: 1
-                  cilium             quay.io/cilium/cilium-service-mesh:v1.11.0-beta.1: 2
+Image versions    cilium             quay.io/cilium/cilium:v1.12.0-rc1: 2
+                  cilium-operator    quay.io/cilium/operator-generic:v1.12.0-rc1: 1
+                  hubble-relay       quay.io/cilium/hubble-relay:v1.12.0-rc1: 1
 ```
 
 Confirm that the image versions for cilium operator, hubble-relay, and cilium are the [latest beta builds](https://github.com/cilium/cilium-service-mesh-beta#image-tags).
```

Comment:
Is this link right? I think we're building from the main repo now so not sure the image tags will be here?

Reply:
No, this PR also updates the image versions in the table.

Reply:
oh yeah you are right. let's ship it.